scalyr_agent/copying_manager.py

```diff
@@ -370,10 +370,14 @@ def __path_in_globs(self, path, path_globs):
 
         return None
 
-    def add_log_config(self, monitor_name, log_config):
+    def add_log_config(self, monitor_name, log_config, force_add=False):
```

Comment:
Part of me wonders if we shouldn't just always have `force_add=True` (and therefore remove it as a parameter).  I know you are trying to limit the scope of this change, but I do wonder if other similar bugs will crop up on paths where `force_add=False`.  Did you take a look to get a sense?

I have to admit, it was not on my radar to handle the case where a docker container is stopped and started again.

I'm a little worried what happens in the case where the container is stopped for a longer period of time and then restarted, while the original log file still remains on disk.  I'm a little worried that we will start copying the log again from the start of the log file, rather than where we left off.  Here's what I am thinking:

1. Docker container stops.
2. Scalyr Agent the log file for removal.
3. (I believe) Once the Scalyr Agent catches up to the end of the current log file, the LogProcessor is removed.
4. (I believe) At some later point, the checkpoint for that log file is removed.
5. Docker container starts again.  The original log file content is still in the docker log file.
6. (I believe) The Scalyr Agent will notice the container running, and add back in the log file to be copied, configured with copying from start (because I think that's how we handle Docker containers).

Can you take a look to see if that is what would happen?  Step 4 probably doesn't matter (if we have a checkpoint state for it or not) since we are going to copy from the start anyway.  But, I wanted to highlight that a checkpoint for a file is removed at some point just in case it does matter.